### PR TITLE
Adding mesh app, volume, service, service-replica commands

### PR DESCRIPTION
**Pull request by rapatchi**
Fixes # .

Changes include:
Adding mesh commands for:
- app
- volume
- service
- service-replica

Verifed:

- [x] Build and test CI passes
- [x] History and readme updated to reflect changes
- [ ] Package version updated according to semantic versioning rules
- [x] Tests modified or added, when applicable
- [x] Updated code owners file, when applicable
- [x] Read the [PR checklist](https://github.com/Azure/service-fabric-cli/wiki/Checklist)

### CODE REVIEW

File: src/sfctl/tests/help_text_test.py [hunk 1]
```diff
@@ -305,6 +305,21 @@ def test_help_documentation(self):
             'sfctl is',
             commands=('command', 'query'))
 
+        self.validate_output(
+            'sfctl mesh app',
+            commands=('delete', 'show'))
+
+        self.validate_output(
```
Comment (jeffj6123) [inline src/sfctl/tests/help_text_test.py]: Shouldnt this be delete, show, get

File: src/sfctl/tests/help_text_test.py [hunk 2]
```diff
@@ -305,6 +305,21 @@ def test_help_documentation(self):
             'sfctl is',
             commands=('command', 'query'))
 
+        self.validate_output(
+            'sfctl mesh app',
```
Comment (jeffj6123) [inline src/sfctl/tests/help_text_test.py]: Shouldnt this be delete, show, get

**Comment by Christina-Kang** [comment]
@rapatchi there are a few things missing which should be added to this change. For example, paths generation tests and release notes. This checklist has more details: [Checklist](https://github.com/Azure/service-fabric-cli/wiki/Checklist) Thanks!

File: src/sfctl/tests/help_text_test.py (hunk 2 above)
Comment (rapatchi) [reply src/sfctl/tests/help_text_test.py]: done

**Pull request merged**